addon/bootstrap.js

```diff
@@ -59,7 +61,59 @@ const appStartupObserver = {
   }
 }
 
+const LibraryButton = {
+  ITEM_ID: "appMenu-library-screenshots",
+
+  init(webExtension) {
+    this._initialized = true;
+    let permissionPages = [...webExtension.extension.permissions].filter(p => (/^https?:\/\//i).test(p));
```

Comment:
Why wouldn't the regex work? `(/^https?:\/\//i).test("http://localhost:10080")` in my JS console returns `true`. :-)